From 776f5107842de11499ad4d749512cf2d58b02b9c Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Wed, 7 Mar 2018 08:27:49 +0800
Subject: [PATCH] Revert "mmc: dw_mmc: Fix the CTO timeout overflow calculation
 for 32-bit systems"

This reverts commit bc6e99f24320fd3e323996756970b848be0f3af3.

Change-Id: I8649faa9e16baa0024030f9f58482840c90fb255
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 drivers/mmc/host/dw_mmc.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/mmc/host/dw_mmc.c b/drivers/mmc/host/dw_mmc.c
index d4fc22974d3d..6e3ea0856e7b 100644
--- a/drivers/mmc/host/dw_mmc.c
+++ b/drivers/mmc/host/dw_mmc.c
@@ -376,8 +376,7 @@ static inline void dw_mci_set_cto(struct dw_mci *host)
 	cto_div = (mci_readl(host, CLKDIV) & 0xff) * 2;
 	if (cto_div == 0)
 		cto_div = 1;
-	cto_ms = DIV_ROUND_UP_ULL(MSEC_PER_SEC * cto_clks * cto_div,
-				  host->bus_hz);
+	cto_ms = DIV_ROUND_UP(MSEC_PER_SEC * cto_clks * cto_div, host->bus_hz);
 
 	/* add a bit spare time */
 	cto_ms += 10;
-- 
2.35.3

